tests/manage/test_dynamic_pvc_accessmodes_with_reclaim_policies.py

```diff
@@ -0,0 +1,375 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import (
+    TimeoutExpiredError, CommandFailed, UnexpectedBehaviour
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pvc import PVC
+from ocs_ci.utility import templating
+from tests import helpers
+from tests.fixtures import (
+    create_rbd_secret, create_cephfs_secret, create_ceph_block_pool,
+    create_project
+)
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_cephfs_secret.__name__,
+    create_ceph_block_pool.__name__,
+    create_project.__name__
```

Comment:
shouldn't we create the project before secret creation?

Reply:
It doesn't matter here because rbd secret, cephfs secret and block pool are being created in `defaults.ROOK_CLUSTER_NAMESPACE` namespace.

Reply:
It shouldn't matter but updated anyways

Reply:
Thanks!